src/wallet/wallet.ts

```diff
@@ -111,8 +111,12 @@ export const generateFlowWallet = async (mnem: string): Promise<Wallet> => {
  * @param mnem mnemonic seed to use
  * @returns wallet
  */
-export const generateBscWallet = async (testnet: boolean, mnem: string): Promise<Wallet> => {
-    return generateEthWallet(testnet, mnem);
+ export const generateBscWallet = async (testnet: boolean, mnem: string): Promise<Wallet> => {
+  return generateEthWallet(testnet, mnem);
+};
+
+export const generateXdcWallet = async (testnet: boolean, mnem: string): Promise<Wallet> => {
+  return generateEthWallet(testnet, mnem);
```

Comment:
XDC derivation path should be 44'/550'/ according to https://github.com/satoshilabs/slips/blob/master/slip-0044.md